### PR TITLE
sof-kernel-log-check: ignore i2c error for DELL CML HDA laptop + i915 Type-C error

### PR DESCRIPTION
**Pull request by RuiqingX**
Fixes: #396

Signed-off-by: RuiqingX <ruiqingx.zhang@intel.com>

### CODE REVIEW

File: tools/sof-kernel-log-check.sh [hunk 1]
```diff
@@ -57,6 +61,15 @@ ignore_str="$ignore_str"'|thermal thermal_zone.*: failed to read out thermal zon
 ignore_str="$ignore_str"'|iwlwifi 0000:00:14\.3: Microcode SW error detected\. Restarting 0x0\.'
 ignore_str="$ignore_str"'|: authentication with ..:..:..:..:..:.. timed out'
 
+# I915, issues reported by sof-test
+# BugLink: https://github.com/thesofproject/sof-test/issues/374
+ignore_str="$ignore_str"'|i915 0000:00:02\.0: \[drm\] ERROR TC cold unblock failed'
+ignore_str="$ignore_str"'|i915 0000:00:02\.0: \[drm\] ERROR TC cold block failed'
```
Comment (marc-hb) [inline tools/sof-kernel-log-check.sh:67]: I have a couple questions about the Type-C errors, please check bug 374

**Review by marc-hb — CHANGES_REQUESTED** [review]
Please move these to a platform specific section below:
```
case "$platform" in
```

**Review by marc-hb — APPROVED** [review]
In the future it would be better to guard more of these with `case $platform`, especially the ones with strings like "DELL0955" in the error message but Rome wasn't built in a day.

**Comment by RuiqingX** [comment]
fixes and closed.

**Comment by RuiqingX** [comment]
It was closed by mistake

**Pull request merged**